pkg/operations/start.go

```diff
@@ -132,12 +132,19 @@ func StartVM(vm *api.VM, debug bool) error {
 		return err
 	}
 
-	// TODO: Enable this when ignite-spawn doesn't write to the file anymore
-	//if err := providers.Client.VMs().Set(vm); err != nil {
-	//	return err
-	//}
+	result, err := providers.Runtime.InspectContainer(containerID)
```

Comment:
You might want to add a comment above that we need to do this differently for CNI; there the IP address should be got from the CNI netns. We can follow that up

Reply:
I'll add that

Reply:
Added in 94f7149.